components/ntp_background_images/browser/view_counter_model.cc

```diff
@@ -75,15 +75,16 @@ void ViewCounterModel::RegisterPageViewForBrandedImages() {
   // view of the branded wallpaper.
   count_to_branded_wallpaper_--;
   if (count_to_branded_wallpaper_ < 0) {
-    // Reset count and increse image index for next time.
+    // Reset count and randomize image index for next time.
```

Comment:
With this change, all users(opted-in or opted-out brave-ads) will see randomized NTP-SI.
However, this PR's title is `randomly choose NTP-SI for users opted-out`?

Reply:
Thanks, I will change the title as yes both opted-in and out of brave ads is random.

Reply:
and I think first image should be randomized also (https://github.com/brave/brave-core/pull/15076/files#diff-01207863c3305317fa117a16b28a07b253f2315aaad04eb258af6645b86aff89R25)?

Reply:
First image should be random, I will fix this first thing tomorrow

Reply:
@simonhong Fixed, thanks